CMakeLists.txt

```diff
@@ -125,17 +125,26 @@ endif()
 # If trilinos is used, it comes with Kokkos so we don't need standalone
 if(NOT HAVE_TRILINOS AND HAVE_KOKKOS)
   if(HAVE_KOKKOS)
-    if(KOKKOS_ROOT_DIR)
+    if (Kokkos_ROOT)
```

Comment:
I'm sort of wondering if this extra branch is still needed. `target_link_libraries(nimble PUBLIC Kokkos::kokkos)` is in theory supposed to set OpenMP flags if kokkos is built with OpenMP and also set the include directories correctly

Reply:
I removed a test that was always true.

I am not sure to understand your comment about OpenMP. I agree with you about the libraries and the include directories. 

I do not believe the CXXFLAGS will be propagated and will contain "-fopenmp". What do you think?

Reply:
If Kokkos is set up properly (which I think it is for more recent version of Kokkos), doing `target_link_libraries(nimble PUBLIC Kokkos::kokkos)` will also set "PUBLIC" compiler flags, such as fopenmp, etc to any target it's linked to. It's similar to how flags are propagated from the libnimble target to the various executables

Reply:
Fixed CMakeLists.txt